docs/extending/extending_form_builder.md

```diff
@@ -7,19 +7,23 @@
     ### Captcha field
 
     The Captcha Form field is based on [Gregwar/CaptchaBundle](https://github.com/Gregwar/CaptchaBundle).
-    
+
     You can customize the field by adding configuration to `config/packages/gregwar_captcha.yaml` under `gregwar_captcha`:
-        
+
     ``` yaml
     gregwar_captcha:
         as_url: true
         width: 150
         invalid_message: Retry.
         reload: true
     ```
-    
+
     For information about available options, see [the bundle's documentation.](https://github.com/Gregwar/CaptchaBundle#options)
 
+    !!! note
+
+        If your installation uses Varnish to manage content cache, you must modify the configuration to avoid issues with the Captcha field. For more information, see [Ensure proper captcha behavior](../guide/http_cache.md#ensure-proper-captcha-behavior).
```

Comment:
If I read the inline review right, @mnocon has seen this also with local symfony httpcache proxy, so maybe safe to say it affects all cache setups? Aka it affects all setups in production basically..

If so we should not state it's limited to Varnish